client_with_ssl.go

```diff
@@ -8,17 +8,16 @@ import (
 	"github.com/hazelcast/hazelcast-go-client/serialization"
 	"github.com/hazelcast/hazelcast-go-client/sql"
 	"github.com/hazelcast/hazelcast-go-client/types"
-	"log"
-	"math/rand"
 	"os"
 	"path/filepath"
+	"reflect"
 	"time"
 )
 
 // This is boilerplate application that configures client to connect Hazelcast Cloud cluster.
 // After successful connection, it runs the uncommented examples.
 //
-// See: https://docs.hazelcast.cloud/docs/go-client
+// See: https://docs.hazelcast.com/cloud/get-started
 func main() {
 	_ = os.Setenv("HZ_CLOUD_COORDINATOR_BASE_URL", "YOUR_DISCOVERY_URL")
```

Comment:
This line is not required anymore.

Reply:
Fixed at https://github.com/hazelcast/hazelcast-cloud-go-sample-client/pull/14/commits/8780cc95bbc6f29f809d266101019e026a081ca3